thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/client/cache/SchemaCacheLoader.java

```diff
@@ -23,6 +23,7 @@
 
   private static final String UTF_8 = "UTF-8";
   private static final String SCHEMA_ENDPOINT = "schemas/";
+  private static final ObjectMapper OBJECT_MAPPER = new ObjectMapper();
```

Comment:
For this use case may be it is ok to initialize ObjectMapper as static field. 
This is how we should initialize non threadsafe objects. 

  private static final ThreadLocal<ObjectMapper> OBJECT_MAPPER_THREAD_LOCAL = new ThreadLocal<ObjectMapper>() {
    @Override
    protected ObjectMapper initialValue() {
      return new ObjectMapper();
    }
  };